### DISCUSSION TITLE
Unable to pass step 6 of getting started - Add to Key Vault access policies

### DISCUSSION THREAD

**Discussion by starykolarz** [polymind-inc/acmebot#700] [Q&A]
I have the following info in Access Policies panel:

Access policies not available.
The access configuration for this key vault is set to role-based access control. To add or manage your access policies, go to the Access control (IAM) page

Probbly because I moved to RBAC as enforced by MS.   

Also config described here doesn't work for me - I don't have a way to add the DNS provider (https://github.com/shibayan/keyvault-acmebot/wiki/DNS-Provider-Configuration#azure-dns)

In the dashboard I get the following error: 

Orchestrator function 'GetCertificates_Orchestrator' failed: The activity function 'GetAllCertificates' failed: "DNS Provider is not configured. Please check the documentation and configure it.". See the function execution logs for additional details.

How to I add appropriate configuration in this case?

**Comment by shlubz** [comment]
I have the same issue after migrating to version 4 and getting the same error.  Noticed none of our certificates automatically renewing and seen in a discussion we had to update the .net to version 4 and point to a new repository for the application.   All my functions are showing enabled and I cant find Acmebot:AzureDns:SubscriptionId anywhere in the function app to see if the subscription ID is set.

**Reply by shlubz** [reply]
The subscriptionID is set so I'm not sure why mine is broken.  Tried restarting the app and doesn't fix.  All the variables are set correctly.
